index.ts

```diff
@@ -1398,6 +1453,13 @@ const crypto: { node?: any; web?: any } = {
   web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
 };
 
+const TAGS = ['BIP0340/challenge', 'BIP0340/aux', 'BIP0340/nonce'] as const;
```

Comment:
Should we use this array in line 1424 etc.?

Reply:
If we switch `taggedHash` to take `string` instead of the enumerated type, yes. With the enumerated type, the strings are a bit more clear, and retain correctness guarantees.

Reply:
lets do it